src/components/Square.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react'
+
+//component
+const Square = ({
+  id,
+  currentTurn,
+  value,
+  disabled,
+  winningSquares,
+  handleClick,
+}: SquareProps) => {
+  const [stateValue, setValue] = useState<string>('')
```

Comment:
```suggestion
  const [stateValue, setStateValue] = useState<string>('')
```

for useState hooks, your mutator should always just be the name of your accessor with a set- prefix